beesdoo_print_label/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Beesdoo Print Label",
     "summary": """Product labels""",
-    "version": "12.0.2.0.0",
+    "version": "12.0.2.0.1",
```

Comment:
it’s the wrong module. it should be `beesdoo_product_label`. 😉

Reply:
ah je suis bien benêt, c'est réparé !